package.json

```diff
@@ -25,8 +25,10 @@
   "author": "Maximilian Hoffmann",
   "license": "MIT",
   "devDependencies": {
-    "fs-extra": "^0.16.5",
     "tap-dot": "^1.0.0",
     "tape": "^3.5.0"
+  },
+  "dependencies": {
+    "npm": "^3.5.1"
```

Comment:
I’m not sure installing `npm` locally is worth the benefit of having the correct `PATH` set up for the hooks. Do you know if this works without problems when the global and local `npm` version differ? Especially version 2 and 3 as they install packages differently.